concent_api/core/tests/test_unit_model.py

```diff
@@ -308,11 +308,6 @@ def test_that_exception_is_raised_when_amount_is_less_then_zero(self):
             self.deposit_claim.clean()
         self.assertIn('amount', exception_info.value.error_dict)
 
-    def test_that_exception_is_raised_when_amount_is_of_wrong_type(self):
-        self.deposit_claim.amount = 5.0
```

Comment:
I would suggest keeping this test but changing the value to a non accepted type. This way we still test one branch of the `if` condition (after doing the comments I have added). @dybi do you have opinion on that? Is it better to remove it?

Reply:
If `DecimalField` validations check that for us automatically, then we don't have to add tests for that. If not, then it would be good to have one.

Reply:
Done